etna/transforms/timestamp/holiday.py

```diff
@@ -83,14 +141,32 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         -------
         :
             pd.DataFrame with added holidays
+
+        Raises
+        ------
+        ValueError:
+            if the frequency is greater than daily and this is a ``binary`` or ``categorical`` mode
+        ValueError:
+            if the frequency is less than daily and this is ``days_count`` mode
```

Comment:
We don't have this error. We have error on unsupported frequencies (which are all except for some limited list).